gr-uhd/lib/usrp_source_impl.cc

```diff
@@ -440,10 +440,14 @@ namespace gr {
     void
     usrp_source_impl::issue_stream_cmd(const ::uhd::stream_cmd_t &cmd)
     {
+#ifdef INCLUDED_UHD_UTILS_MSG_TASK_HPP
```

Comment:
This is hacky, but fine. However, please add a comment to this line, because it's counter-intuitive ("this define was added in 3.6, the same time as this API call"). @jmcorgan assuming @noc0lour adds this fix, I'm fine with this.